apps/src/gamelab/animationListModule.js

```diff
@@ -242,6 +242,19 @@ export function setInitialAnimationList(serializedAnimationList) {
     }
   }
 
+  // If animations have the same name, rename one.
+  let numberAnimations = serializedAnimationList.orderedKeys.length;
+  for (let i = 0; i < numberAnimations; i++) {
+    let key = serializedAnimationList.orderedKeys[i];
+    let name = serializedAnimationList.propsByKey[key].name;
+    for (let j = i + 1; j < numberAnimations; j++) {
+      let otherKey = serializedAnimationList.orderedKeys[j];
+      if (name === serializedAnimationList.propsByKey[otherKey].name) {
+        serializedAnimationList.propsByKey[key].name = generateAnimationName(name, serializedAnimationList.propsByKey);
+      }
+    }
+  }
+
```

Comment:
Not necessarily something that needs to happen in this PR, but I setInitialAnimationList is getting longish. I wonder if it makes sense to split this logic, and the frameRates/frameDelays logic into separate methods (which might also then make unit tests simpler).